charts/minecraft-bedrock/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: minecraft-bedrock
-version: 1.1.0
-appVersion: 1.14.4
+version: 1.1.1
+appVersion: 1.16.1
```

Comment:
appVersion is always a strange animal IMHO. Perhaps just `1.16` to track it fairly close

Reply:
Works for me, will change this too!